### PR TITLE
Implement pagination and refactor front-end layout

### PR DESCRIPTION
**Pull request by eric1052**

### CODE REVIEW

File: client/dist/style.css [hunk 1]
```diff
@@ -195,37 +203,84 @@
...
+  font-weight: normal;
+}
+
+.page-button:focus {
+  outline: 0;
+}
+
+.page-button:hover:not(.active) {
+  text-decoration: underline;
+}
```
Comment (Implausibility) [inline client/dist/style.css:286]: Newline.

File: database/seeder.js [hunk 2]
```diff
@@ -30,5 +28,5 @@ const generateReviews = () => {
 };
 
 for (let i = 0; i < 100; i += 1) {
-  db.Room.collection.insertOne({ _id: i + 1, reviews: generateReviews() });
+  db.Room.collection.insertOne({ id: i + 1, reviews: generateReviews() });
```
Comment (Implausibility) [inline database/seeder.js:31]: Much appreciated!

File: client/src/components/Pagination.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      'font-weight': 'bold',
+    });
+    for (let i = 0; i < pages.length; i += 1) {
+      if (pages[i] === Number(currentPage)) {
+        continue;
+      } else {
+        $(`.page-button-${pages[i]}`).css({
+          color: 'grey',
+          border: 'none',
+          'font-weight': 'normal',
```
Comment (Implausibility) [inline client/src/components/Pagination.jsx:26]: I'm not 100% clear on what's happening here. Are you rendering all the page numbers of the pages other than the one the user is currently on?

File: client/src/components/Header.jsx [hunk 4]
```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import PropTypes from 'prop-types';
```
Comment (Implausibility) [inline client/src/components/Header.jsx:2]: Why the move away from PropTypes? They seem a useful check to ensure the data is there.

File: client/src/components/Pagination.jsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import $ from 'jquery';
+
+const Pagination = ({ numReviews, paginate }) => {
+  const pages = [];
+  const numPages = Math.ceil(numReviews / 7);
+  for (let i = 1; i <= numPages; i += 1) {
+    pages.push(i);
+  }
```
Comment (Implausibility) [inline client/src/components/Pagination.jsx:9]: I suspect you're going to need either a useState hook or to turn this into a class component to have one page formatted at the start.

There may be an easier way, like always initially rendering page 1 that way in the beginning, but I haven't used jQuery to apply CSS like you are. There may be a way to do it with that technique that I'm just not familiar with.

File: client/src/components/Pagination.jsx (hunk 3 above)
Comment (eric1052) [reply client/src/components/Pagination.jsx:26]: Sort of—I'm styling all the pages except the one the user is currently on with a particular style. Definitely not the most efficient way though; I'm hoping to refactor it if time allows.

File: client/src/components/Header.jsx (hunk 4 above)
Comment (eric1052) [reply client/src/components/Header.jsx:2]: Ah yes, I'm definitely going to bring it back. I was getting all these weird errors with prop types so I removed it temporarily.

File: client/src/components/Pagination.jsx (hunk 5 above)
Comment (eric1052) [reply client/src/components/Pagination.jsx:9]: Thanks, I'll look into the different options!

**Pull request merged**